autofs-5.1.0 - fix compile error in defaults.c

From: Ian Kent <raven@themaw.net>

Puzzling, suddenly I'm getting a compile error for defaults.c due
to a lack of including stdarg.h.
diff --git a/lib/defaults.c b/lib/defaults.c
index 2b03ea2..d29a976 100644
--- lib/defaults.c
+++ lib/defaults.c
@@ -20,6 +20,7 @@
 #include <string.h>
 #include <sys/utsname.h>
 #include <sys/stat.h>
+#include <stdarg.h>
 
 #include "config.h"
 #include "list.h"
